client/assets/javascripts/modules_new/DetailsExpander.js

```diff
@@ -1,27 +1,32 @@
 const DetailsExpander = {
   init: function (nodes) {
```

Comment:
Yeah good shout. It took me a while to figure out that nodes in this instance was just referring to the document, so worth renaming to prevent confusion!